src/Blade.php

```diff
@@ -113,21 +113,13 @@ public function __call(string $method, array $params)
 
     protected function setupContainer(array $viewPaths, string $cachePath)
     {
-        $this->container->bindIf('files', function () {
-            return new Filesystem;
-        }, true);
-
-        $this->container->bindIf('events', function () {
-            return new Dispatcher;
-        }, true);
-
-        $this->container->bindIf('config', function () use ($viewPaths, $cachePath) {
-            return [
-                'view.paths' => $viewPaths,
-                'view.compiled' => $cachePath,
-            ];
-        }, true);
-        
+        $this->container->singleton('files', fn () => new Filesystem);
```

Comment:
I guess this is the breaking change not making it compatible with previous versions. Unless you would use something like a `function_exists` to switch between the two. But it looks like it's a completely new interface as well.

Reply:
I switched to use `bindIf` as in the current version.